### ISSUE TITLE
OpenJDK 1.8.0-1 Object reference not set to an instance of an object

### ISSUE THREAD

**Issue by YounesCheikh** [Redth/dotnet-maui-check#29]
Hi, 

I'm running the check for the first time on Win 10 and got the following error with version '0.4.2'

```
────────────────────────────────────────» Synchronizing configuration... ok
» Scheduling appointments... ok

> OpenJDK 1.8.0-1 Checkup...

────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────¤ Recommendation: Install OpenJDK11
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
! Attempt to fix? [y/n] (y): y
» Attempting to fix: OpenJDK 1.8.0-1
  Installing Download and Install Microsoft OpenJDK 11...
**Fix failed - Object reference not set to an instance of an object.**
```

is this normal? anybody encountered the same issue ?
Thanks in advance

**Comment by burhanf** [comment]
I am having this issue as well

**Comment by Redth** [comment]
Can you run this with `--verbose` and share the output?

**Comment by burhanf** [comment]
`> OpenJDK 1.8.0-1 Checkup...
ValidateAndroidSdkLocation: ``, result=False
ValidateAndroidSdkLocation: ``, result=False
Looking for Android SDK...
ValidateJavaSdkLocation: ``, result=False
ValidateJavaSdkLocation: ``, result=False
ValidateJavaSdkLocation: `C:\Program Files\Java\jdk1.8.0_281`, result=True
ValidateAndroidNdkLocation: ``, result=False
ValidateAndroidNdkLocation: ``, result=False
Looking for Android NDK...
ValidateAndroidNdkLocation: `C:\Users\Burhan\AppData\Local\Android\Sdk`, result=False
ValidateAndroidNdkLocation: `C:\Users\Burhan\AppData\Local\Android\Sdk`, result=False
Looking for Android SDK...
System.InvalidOperationException: Could not determine Android SDK location. Please provide `androidSdkPath`.
  at Xamarin.Android.Tools.AndroidSdkInfo..ctor(Action`2 logger, String androidSdkPath, String androidNdkPath, String
     javaSdkPath)
  at DotNetCheck.Checkups.OpenJdkCheckup.Examine(SharedState history) in
     /Users/runner/work/dotnet-maui-check/dotnet-maui-check/MauiCheck/Checkups/OpenJdkCheckup.cs:53
javac 1.8.0_281
javac 1.8.0_281

────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────☼ Recommendation: Install OpenJDK11
────────────────────────────────────────────────────────────────────────────────────────────────────────────────────────
! Attempt to fix? [y/n] (y): y
» Attempting to fix: OpenJDK 1.8.0-1
  Installing Download and Install Microsoft OpenJDK 11...
System.NullReferenceException: Object reference not set to an instance of an object.
  at DotNetCheck.Solutions.BootsSolution.Implement(SharedState sharedState, CancellationToken cancellationToken) in
     /Users/runner/work/dotnet-maui-check/dotnet-maui-check/MauiCheck/Solutions/BootsSolution.cs:24
  at DotNetCheck.Cli.CheckCommand.ExecuteAsync(CommandContext context, CheckSettings settings) in
     /Users/runner/work/dotnet-maui-check/dotnet-maui-check/MauiCheck/CheckCommand.cs:216
Fix failed - Object reference not set to an instance of an object.`

**Comment by burhanf** [comment]
I did 'maui-check --dev' and it seems to have downloaded and installed the JDK and other missing components now
